test/python/opflow/test_op_construction.py

```diff
@@ -963,14 +963,14 @@ def test_op_parameters(self):
         qc.h(0)
         qc.x(1)
 
-        l = Parameter("λ")
-        op = PrimitiveOp(qc, coeff=l)
+        _lambda = Parameter("λ")
+        op = PrimitiveOp(qc, coeff=_lambda)
```

Comment:
Python convention is to suffix `_` on keywords rather than prefix, since prefixing is used to mean "explicitly ignored/private".

Reply:
Fixed in 148f28f90440ef6c98e66a142d754908de741f92